events/events.go

```diff
@@ -55,6 +55,8 @@ type Config struct {
 	//
 	// If the passed in context object already has an earlier deadline set,
 	// that deadline will be respected instead.
+	// But if the passed in context is already canceld,
```

Comment:
Small typo, should be “ cancelled”

Reply:
The variable is called `context.Canceled`: https://pkg.go.dev/context?tab=doc#pkg-variables

Reply:
technically both `cancelled` and `canceled` are correct in English, I think.

Reply:
Ya, I guess they are...

:til:

(I’ve probably written it both ways...)

Reply:
Well, _this_ one still has a small type 😅

“canceld”

Reply:
:shakes-fist: fixed.